src/components/forums/forum/ForumContent.tsx

```diff
@@ -87,18 +89,17 @@ export function ForumContent(props: ForumContentProps) {
 
   const addAccessToken = async () => {
     setAddingAccessToken(true);
-    try {
-      const restriction = pubkeysToRestriction(
-        newForumAccessToken,
-        isSuccess(forumData.restriction) ? forumData.restriction : undefined
-      );
-      const currentIds = restrictionListToString(restriction);
-
-      const tx = await forumObject.setForumPostRestriction(
-        forumData.collectionId,
-        restriction
-      );
+    const restriction = pubkeysToRestriction(
```

Comment:
this `pubkeysToRestriction` and `restrictionListToString` functions don't have try catch blocks, I think it will cause a crash here if there's an error. Should probably add it in the util functions here